File: Selenium/Activities/Activity17.java
package example;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Select;

public class Activity17 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/selects");
		System.out.println("Title of the page is: " + driver.getTitle());
		WebElement ele = driver.findElement(By.xpath("//select [contains(@class,'h-80')]"));
		Select select=new Select(ele);
		select.selectByVisibleText("HTML");
		select.selectByIndex(4);
		select.selectByIndex(5);
		select.selectByIndex(6);
		select.selectByValue("nodejs");
		select.deselectByIndex(5);
		driver.quit();

	}

}

File: Selenium/Activities/Activity17.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.common.by import By
import time
from selenium.webdriver.support.select import Select

service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://training-support.net/selenium/selects")
    print("Page Title :- " + driver.title)
    sel=Select(driver.find_element(By.ID,"single-select"))
    sel.select_by_visible_text("Option 2")
    sel.select_by_index(2)
    sel.select_by_value("4")
    opt=sel.options
    
    for option in opt:
        print(option.text)

    driver.close()